16/1/src/game.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useState } from 'react';
+import _ from 'lodash';
+
+export default function Game(props) {
+  const [points,setPoints] = useState(0);
+  return (
+    <div>
+      <Points points={points}/>
+      <GameArea setPoints={setPoints}/>
+    </div>
+  );
+}
+
+
+function GameArea({setPoints}) {
+  const [selected,setSelected] = useState(0);
+  
+  const styleBox = {
```

Comment:
I would define constant style objects outside the component code - easiest is to put it  in the same file above the function.

the rational is that objects defined within the component are created over and over again every time the function runs (i.e. every time the component is rendered).

Reply:
And yeah I think you took the code from my example so this is a comment to myself too :)